src/core-services/catalog/utils/getCatalogProductMedia.js

```diff
@@ -27,6 +29,7 @@ export default async function getCatalogProductMedia(productId, collections) {
       const { priority, productId: prodId, variantId } = metadata || {};
 
       return {
+        _id: encodeOpaqueId("reaction/mediaRecord", media._id),
```

Comment:
We always encode IDs in the resolvers because some internal calls expect IDs to come back still internal. In `src/core/graphql/resolvers/index.js`, you can add a resolver for `ImageInfo._id`:

```js
import encodeOpaqueId from "@reactioncommerce/api-utils/encodeOpaqueId.js";

export default {
  ImageInfo: {
    _id: (node) => encodeOpaqueId("reaction/mediaRecord", node._id)
  },
  // other stuff
};
```

Plus then you can include the `_id` but remove the encoding from all other files that have it.